nservicebus/transactional-session/index.md

```diff
@@ -172,24 +142,29 @@ Internally, the transactional session doesn't use a single transaction that span
 ### Phase 1
 
 1. The user opens a transactional session.
-2. A set of `PendingOperations` is initialized and collects the message operations.
+2. A set of `PendingTransportOperations` is initialized and collects the message operations.
 3. A transaction is started on the storage seam.
-4. The user can execute any required message operations using the transactional session.
-5. The user can store any data using the persistence-specific session, which is accessible through the transactional session.
-6. When all operations are registered, the user calls `Commit` on the transactional session.
-7. A control message to complete the transaction is dispatched to the local queue. The control message is independent of the message operations and is not stored in the outbox record.
-8. The message operations are converted and stored into an outbox record.
+4. The storage returns an open transaction.
+5. The user can execute any required message operations using the transactional session.
+6. Transport operations are captured by the `PendingTransportOperations`
+7. The user can store any data using the persistence-specific session, which is accessible through the transactional session.
+8. When all operations are registered, the user calls `Commit` on the transactional session.
+9. A control message to complete the transaction is dispatched to the local queue. The control message is independent of the message operations and is not stored in the outbox record.
+10. The message operations are converted into an outbox record.
+11. The outbox record is returned to the transactional session
+12. The outbox record is saved to the storage seam.
 9. The transaction is committed, and the outbox record and business data modifications are stored atomically.
```

Comment:
Documentation Review: It still renders as 13 because that's how markdown works, but did anyone notice that this list now goes 12 -> 9?

Reply:
Oops. I did not notice. Fixed in https://github.com/Particular/docs.particular.net/commit/eff2229b1751f8b1bc1a4c1e3bca2500379fa5d0